holoviews/core/dimension.py

```diff
@@ -1077,13 +1077,25 @@ def __unicode__(self):
         return unicode(PrettyPrinter.pprint(self))
 
 
-
     def __call__(self, options=None, **kwargs):
+        self.warning('Use opts method instead')
```

Comment:
This was mostly an experiment and having actually seen the results, I agree.

> I'd even say that we should have one release where the only thing nudging users towards using .opts is consistent usage of .opts throughout our docs.

That is a start though can we add this warning for the release cycle following that? I'll remove this warning from this PR.